### DISCUSSION TITLE
Shadow borders on menus and tabs previews

### DISCUSSION THREAD

**Discussion by nayrazag** [Alex313031/thorium-legacy#51] [Q&A]
![1](https://github.com/Alex313031/thorium-win7/assets/96781670/11617836-ae32-42e0-867f-38c857d26bc4)
Hello. How to get rid of this shadow borders around tabs preview and any menus in Thorium?

**Comment by Alex313031** [comment]
This is due to Supermium changes that are partially incompatible with the new CR23 UI. Either disable the CR23 flags or try to live with it is all I can say.

**Reply by Marek33** [reply]
Unfortunately, that doesn't fix it for me. The only fix, to me, is disabling **graphics acceleration**, which makes stuff like Google Maps or Windy, slower. I'm not using any of the **Chrome Refresh 2023** flags.

Thou, I never had this issue, in the Supermium which makes me wonder if it isn't because the Thorium, has a different name for the exe file.
